README.md

```diff
@@ -19,7 +19,7 @@ Update the following fields in the `user.yaml` and save the file
    `sh setup_case.sh -f ../cases/3dvar_only_exp.yaml workflowtest001`
    
```

Comment:
Just add a clarification, so we do not have to come back, something like:

In this example, the 3dvar data assimilation scheme (3dvar_only_exp.yaml) is used.

Reply:
Cleaned up and added a (very) brief description of what the setup example does ...